user-service/sls-resources.yml

```diff
@@ -1,11 +1,86 @@
+slicUserPool:
```

Comment:
The user pool in the user-service and the authorizer usage should be documented in the README and arch diagram